packages/web/src/components/Button/_theme.scss

```diff
@@ -3,9 +3,10 @@
 $padding-x: tokens.$space-7;
 $padding-y: tokens.$space-4;
 $font-family: tokens.$font-family-default;
+$font-weight: tokens.$font-weight-semibold;
 $font-size: tokens.$font-size-base;
-$line-height: tokens.$line-height-default;
-$border-width: 1px;
+$line-height: tokens.$line-height-small;
```

Comment:
According to Figma, buttons should be 40 px tall:

<img width="118" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/144249679-4661cb5f-eca5-4d90-880c-a2dca039ed72.png">

vs.

<img width="401" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/144249622-e7634269-8cbd-461b-af9f-3e0556bd66ba.png">

It seems that the sum of `line-height`, `border` and `padding` is smaller than we need…

Reply:
<img width="209" alt="obrazek" src="https://user-images.githubusercontent.com/5614085/144414314-569b0604-9409-41c6-9e93-d928007a8c10.png">

🤔

Reply:
Let's solve it in a separate PR, we cannot let it block the merge of other changes in this PR.